homeassistant/components/zwave_js/discovery.py

```diff
@@ -362,6 +374,24 @@ class ZWaveDiscoverySchema:
             type={"number"},
         ),
     ),
+    # cover
+    # motorized barriers
+    ZWaveDiscoverySchema(
+        platform="cover",
+        hint="motorized_barrier",
+        primary_value=ZWaveValueDiscoverySchema(
+            command_class={CommandClass.BARRIER_OPERATOR},
+            property={"currentState"},
+            type={"number"},
+        ),
+        required_values=[
+            ZWaveValueDiscoverySchema(
+                command_class={CommandClass.BARRIER_OPERATOR},
+                property={"targetState"},
+                type={"number"},
+            ),
+        ],
+    ),
```

Comment:
Removed the check for device classes. This allows the cover to support node-zwave-js v6.2.0 and any other potential motorized barrier types that we don't know about. The functionality isn't dependent on the types, only the barrier class.

There are some specific types that could result in different HA device classes (e.g. DEVICE_CLASS_GATE), but I haven't seen this kind of device yet, so just assume garage.